lib/membrane/compositor_element.ex

```diff
@@ -6,12 +6,18 @@ defmodule Membrane.VideoCompositor.CompositorElement do
   """
 
   use Membrane.Filter
+  alias Membrane.Buffer
   alias Membrane.RawVideo
   alias Membrane.VideoCompositor.Wgpu
 
   def_options caps: [
-                type: RawVideo,
+                spec: RawVideo.t(),
                 description: "Struct with video width, height, framerate and pixel format."
+              ],
+              live: [
+                spec: boolean(),
+                description: "Set the compositor to live mode",
+                default: false
               ]
```

Comment:
Does `live` option trigger frames dropping? I think it should be better described.